news/2023/2023-05-04-osu-originals-osu-taiko-world-cup-2023.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: "osu! originals - osu!taiko World Cup 2023"
+date: 2023-05-04 11:00:00 +0000
+---
+
+Japan may have won the [osu!taiko World Cup](/wiki/Tournaments/TWC) for the 10th year in a row, but this is the first TWC with **5 brand-new songs** created specifically for the tournament!
+
+![](/wiki/shared/news/banners/featured-artist.jpg)
+
+We call those songs [**osu! originals**](/wiki/osu!_originals) because... they're original songs made for osu!. While our names lack creativity, the creativity in the new songs from [2023's osu!taiko World Cup](/wiki/Tournaments/TWC/2023) makes up for it.
+
+Check out the new tunes for yourself:
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="LINK" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### Toromaru - Sleight of Hand
+
+High-tempo jazz is [**Toromaru**](https://osu.ppy.sh/beatmaps/artists/279)'s specialty, and it perfectly suits the rhythmic complexity that osu!taiko needs.
+
+[Sleight of Hand](https://osu.ppy.sh/beatmapsets/1972518) mapped by [Nurend](https://osu.ppy.sh/users/9905079) slotted into the [Semifinals map pool](/wiki/Tournaments/TWC/2023#semifinals) as a no-mod pick, and led to *two* matches that were decided by less than 20,000 score difference!
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### LV.4 feat. Mawata Spica - Vortex of Sorrow
+
+As a master of hardcore and trance, [**LV.4**](https://osu.ppy.sh/beatmaps/artists/214) ensured that anyone listening to *Vortex of Sorrow* would be immersed in endlessly pounding bass and classic synth melodies. 
+
+[This beatmap](https://osu.ppy.sh/beatmapsets/1980701) by [uone](https://osu.ppy.sh/users/5321719) acted as a freemod pick in the [Grand Finals map pool](/wiki/Tournaments/TWC/2023#grand-finals). If you think you can handle peak osu!taiko gameplay, give it a try.
+
+<audio controls>
+    <source src="LINK" type="audio/mpeg">
+</audio>
+
+### A? - Umami Blaze
```

Comment:
shouldn't this be "Aquestion - Umami Blaze" like in the [linked beatmap](https://osu.ppy.sh/beatmapsets/1976814)?

please see https://github.com/ppy/osu-wiki/pull/9242#discussion_r1181083428 and https://discord.com/channels/188630481301012481/218677502141399041/1101886800231936130 and confirm ty

Reply:
![](https://i.imgur.com/CiuD6Td.png)